logging/logrus/registry.go

```diff
@@ -16,33 +16,35 @@ package logrus
 
 import (
 	"fmt"
-	"sync"
 
-	"github.com/ligato/cn-infra/logging"
 	"github.com/Sirupsen/logrus"
+	"github.com/ligato/cn-infra/logging"
+	"sync/atomic"
 )
 
 // NewLogRegistry is a constructor
 func NewLogRegistry() logging.Registry {
-	registry := &logRegistry{mapping: map[string]*Logger{}}
+	registry := &logRegistry{}
+	// Init mapping in loggers
+	registry.loggers.Store(make(map[string]*Logger))
```

Comment:
please, check again this implementation (see the comments about the logger itself below)